charts/tornjak/templates/server-statefulset.tpl

```diff
@@ -92,13 +73,42 @@ spec:
             - "--shallow"
         {{- end }}
 
+      {{- if .Values.tornjak }}
+      {{- if .Values.tornjak.config }}
+      {{- if .Values.tornjak.config.separateFrontend }}
       - name: tornjak-backend
         image: {{ .Values.tornjak.config.backend.img }}:{{ .Values.tornjak.config.version }}
+      {{- else }}
+      - name: tornjak
+        image: {{ .Values.tornjak.config.img }}:{{ .Values.tornjak.config.version }}
+
+        env:
+
+        {{- if .Values.tornjak.config.frontend }}
+        
+        {{- if .Values.tornjak.config.enableUserMgmt }}
+        {{- if .Values.tornjak.config.frontend.authServerURL }}
+        - name: REACT_APP_AUTH_SERVER_URI
+          value: {{ .Values.tornjak.config.frontend.authServerURL }}
+        {{- end }}
+        {{- end }}
+
+        {{- if .Values.tornjak.config.frontend.apiServerURL }}
+        - name: REACT_APP_API_SERVER_URI
+          value: {{ .Values.tornjak.config.frontend.apiServerURL }}
+        {{- end }}
```

Comment:
this should always be specified - does it matter we have the "if" here?

Reply:
Good point. In fact, there are several values that are required, and they should be enforced. We should have a separate issue just to clean them all up. This specific case is covered by #167 PR.